Testsuite: kn.dmytro_grazhevskiy.usermanagement.gui.MainFrameTest
Tests run: 5, Failures: 5, Errors: 0, Skipped: 0, Time elapsed: 1,212 sec

Testcase: testEditUser took 0,328 sec
	FAILED
mockUserDao: findAll() was expected but not called
junit.framework.AssertionFailedError: mockUserDao: findAll() was expected but not called
	at com.mockobjects.dynamic.Mock.verify(Mock.java:107)
	at kn.dmytro_grazhevskiy.usermanagement.gui.MainFrameTest.tearDown(Unknown Source)
	at junit.extensions.jfcunit.JFCTestCase$2.run(JFCTestCase.java:334)
	at junit.extensions.jfcunit.JFCTestCase$3.run(JFCTestCase.java:478)
	at java.lang.Thread.run(Unknown Source)

Testcase: testAddUser took 0,375 sec
	FAILED
mockUserDao: findAll() was expected but not called
junit.framework.AssertionFailedError: mockUserDao: findAll() was expected but not called
	at com.mockobjects.dynamic.Mock.verify(Mock.java:107)
	at kn.dmytro_grazhevskiy.usermanagement.gui.MainFrameTest.tearDown(Unknown Source)
	at junit.extensions.jfcunit.JFCTestCase$2.run(JFCTestCase.java:334)
	at junit.extensions.jfcunit.JFCTestCase$3.run(JFCTestCase.java:478)
	at java.lang.Thread.run(Unknown Source)

Testcase: testBrowseControls took 0,151 sec
	FAILED
mockUserDao: findAll() was expected but not called
junit.framework.AssertionFailedError: mockUserDao: findAll() was expected but not called
	at com.mockobjects.dynamic.Mock.verify(Mock.java:107)
	at kn.dmytro_grazhevskiy.usermanagement.gui.MainFrameTest.tearDown(Unknown Source)
	at junit.extensions.jfcunit.JFCTestCase$2.run(JFCTestCase.java:334)
	at junit.extensions.jfcunit.JFCTestCase$3.run(JFCTestCase.java:478)
	at java.lang.Thread.run(Unknown Source)

Testcase: testDeleteUser took 0,15 sec
	FAILED
mockUserDao: findAll() was expected but not called
junit.framework.AssertionFailedError: mockUserDao: findAll() was expected but not called
	at com.mockobjects.dynamic.Mock.verify(Mock.java:107)
	at kn.dmytro_grazhevskiy.usermanagement.gui.MainFrameTest.tearDown(Unknown Source)
	at junit.extensions.jfcunit.JFCTestCase$2.run(JFCTestCase.java:334)
	at junit.extensions.jfcunit.JFCTestCase$3.run(JFCTestCase.java:478)
	at java.lang.Thread.run(Unknown Source)

Testcase: testDetailsUser took 0,17 sec
	FAILED
mockUserDao: findAll() was expected but not called
junit.framework.AssertionFailedError: mockUserDao: findAll() was expected but not called
	at com.mockobjects.dynamic.Mock.verify(Mock.java:107)
	at kn.dmytro_grazhevskiy.usermanagement.gui.MainFrameTest.tearDown(Unknown Source)
	at junit.extensions.jfcunit.JFCTestCase$2.run(JFCTestCase.java:334)
	at junit.extensions.jfcunit.JFCTestCase$3.run(JFCTestCase.java:478)
	at java.lang.Thread.run(Unknown Source)

